plugins/queue/providers/kafka/KafkaPlugin.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * @package plugins.kafka
+ */
+class KafkaPlugin extends KalturaPlugin implements IKalturaPending, IKalturaObjectLoader, IKalturaQueuePlugin, IKalturaEnumerator
+{
+	const PLUGIN_NAME = 'kafka';
+	const QUEUE_PLUGIN_NAME = 'queue';
+	const QUEUE_PLUGIN_VERSION_MAJOR = 1;
+	const QUEUE_PLUGIN_VERSION_MINOR = 0;
+	const QUEUE_PLUGIN_VERSION_BUILD = 0;
+	
+	/*
+	 * (non-PHPdoc)
+	 * @see IKalturaPlugin::getPluginName()
+	 */
+	public static function getPluginName()
+	{
+		return self::PLUGIN_NAME;
+	}
+	
+	/*
+	 * (non-PHPdoc)
+	 * @see IKalturaObjectLoader::getObjectClass()
+	 */
+	public static function getObjectClass($baseClass, $enumValue)
+	{
+		if ($baseClass == 'QueueProvider' && $enumValue == self::getKafakaQueueProviderTypeCoreValue(KafkaProviderType::KAFKA)) {
+			if (!kConf::hasMap('kafka')) {
+				throw new kCoreException("Kafka configuration file (kafka.ini) wasn't found!");
+			}
+			
+			return 'KafkaProvider';
+		}
+	}
+	
+	/*
+	 * (non-PHPdoc)
+	 * @see IKalturaObjectLoader::loadObject()
+	 */
+	public static function loadObject($baseClass, $enumValue, array $constructorArgs = null)
+	{
+		if ($baseClass == 'QueueProvider' && $enumValue == self::getKafakaQueueProviderTypeCoreValue(KafkaProviderType::KAFKA)) {
+			if (!kConf::hasMap('kafka')) {
```

Comment:
fix indentation and use defined map name

Reply:
Done.